### PR TITLE
feat: add test instructions, format readme

### PR DESCRIPTION
**Pull request by laszlokr**

### CODE REVIEW

File: sale_blanket_order_readonly_states/tests/TEST_INSTRUCTIONS.rst
Comment (janikvonrotz) [inline sale_blanket_order_readonly_states/tests/TEST_INSTRUCTIONS.rst:1]: What changes can be made? Can you write down a particular example.

**Review by janikvonrotz — CHANGES_REQUESTED** [review]

File: sale_blanket_order_readonly_states/tests/TEST_INSTRUCTIONS.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- Open Sales>Order>Blanket Orders
+- Create a new Blanket Order
+- With the module installed, changes can still be made to some fields until a Sales Order is issued
```
Comment (janikvonrotz) [inline sale_blanket_order_readonly_states/tests/TEST_INSTRUCTIONS.rst]: Can you remove "With the module installed, changes can still be made to some fields until a Sales Order is issued"

This is a observation and not a test action.

**Review by janikvonrotz — CHANGES_REQUESTED** [review]

File: sale_blanket_order_readonly_states/tests/TEST_INSTRUCTIONS.rst
Comment (janikvonrotz) [inline sale_blanket_order_readonly_states/tests/TEST_INSTRUCTIONS.rst:1]: The TEST_INSTRUCTIONS are meant to be a functional verification and not a user manual nor a feature description of the module. So writing "To verify, ..." does not make sense. Here is my version:

- Open Sales > Order > Blanket Orders
- Create a new blanket order and confirm it
- Change validity date of the order

**Review by janikvonrotz — CHANGES_REQUESTED** [review]

File: sale_blanket_order_readonly_states/tests/TEST_INSTRUCTIONS.rst
Comment (laszlokr) [reply sale_blanket_order_readonly_states/tests/TEST_INSTRUCTIONS.rst:1]: @janikvonrotz got it. I update the test intructions with your version and will try to follow similar format

**Review by janikvonrotz — APPROVED** [review]

**Pull request merged**